### PR TITLE
Changes the z-index of the different parts of the progress view.

### PR DESCRIPTION
**Pull request by eireland**
Refactors the header to only show one header when in feedback and response detail view.
Updates cypress test to reflect the changes.

### CODE REVIEW

File: js/containers/portal-dashboard/portal-dashboard-app.tsx [hunk 1]
```diff
@@ -279,22 +279,23 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
...
-        userName={userName}
-        setCompact={headerViewMode === "ProgressDashboard" ? setCompactReport : undefined}
-        setHideFeedbackBadges={headerViewMode === "ProgressDashboard" ? setHideFeedbackBadges : undefined}
-        assignmentName={assignmentName}
-        trackEvent={trackEvent}
-        setDashboardViewMode={this.setDashboardViewMode}
-        viewMode={viewMode}
-        colorTheme={color}
-      />
+      sequenceTree && <Header
```
Comment (emcelroy) [inline js/containers/portal-dashboard/portal-dashboard-app.tsx:289]: I'm not 100% sure about this particular instance, but I think if what comes after `&&` is made up of multiple lines, our convention is to have it start on a new line, as in line 175 above, right?

**Review by emcelroy — APPROVED** [review]
This looks good and worked as expected when I tested it out. I just left one inline comment/question about how one block of code is formatted.

**Comment by mklewandowski** [comment]
What were the changes in the portal dash app for?  When I tested locally last week, I only needed two lines of CSS changes to fix the z-order.

**Comment by eireland** [comment]
Adding the different z-index was affecting the z-index in the response/feedback view.

**Pull request merged**